src/main/webapp/dev/js/common/updateCharLeft.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Updates the number of characters left in the text area
+ * @param textArea - Text area for which the number of characters are to be counted
+ * @param letterCountId - Id of Label to display length of text area
+ */
+function updateCharLeftCount(textArea, letterCountId) {
+    const letterCountArea = $(`#${letterCountId}`);
+    const maxLength = textArea.attr('maxLength');
+    const charLength = textArea.val().length;
+    letterCountArea.text(maxLength - charLength);
+}
+/**
+ * Create the div to display the number of characters left in textArea
+ * @param textArea - Text area for which char are to be counted
```

Comment:
Let's document the parameters exactly the same as `computeRemainingCharactersCount`.